shared/src/commonMain/kotlin/org/hyperskill/app/progresses/view/ProgressScreenViewState.kt

```diff
@@ -0,0 +1,52 @@
+package org.hyperskill.app.progresses.view
+
+import org.hyperskill.app.projects.domain.model.ProjectLevel
+
+data class ProgressScreenViewState(
```

Comment:
Is it possible to extend `ProgressScreenFeature` here? I don't like that the view state is out of the scope of the feature.
I mean smth like that `ProgressScreenFeature.ViewState`

Reply:
@ivan-magda 
No, in Kotlin it's prohibited to create extension classes. Only extension functions are allowed.

Reply:
I moved view state to separate file because it's too big to store it in feature